src/web/mod.rs

```diff
@@ -516,6 +536,17 @@ mod test {
     extern crate env_logger;
     use super::*;
 
+    #[test]
+    fn test_index_returns_success() {
+        crate::test::with_database(|db| {
+            crate::test::with_frontend(db, |web| {
```

Comment:
It might be nice to have `with_frontend()` take an `Option<TestDatabase>` so we don't have patterns like this calling `with_database` then `with_frontend` all over the place.

Reply:
I refactored this away in a later commit.